client/my-sites/importer/newsletter/subscribers/step-initial.tsx

```diff
@@ -73,7 +72,7 @@ export default function StepInitial( {
 			</p>
 
 			<Button
-				href={ `https://${ normalizeFromSite( fromSite ) }/publish/subscribers` }
+				href={ `${ normalizeFromSite( fromSite ) }/publish/subscribers` }
```

Comment:
It seems `normalizeFromSite`  already returns a slash. This gives 404 because I'm getting `//publish/subscribers`

```suggestion
				href={ `${ normalizeFromSite( fromSite ) }publish/subscribers` }
```

Reply:
Oh, haha. normalizeFromSite should strip the `/` at the end because if you 'testfoo.substack.com' it works